content/en/tracing/guide/ingestion_sampling_use_cases.md

```diff
@@ -0,0 +1,153 @@
+
+
+## Determining which ingestion mechanisms are used
+
+To identify what ingestion mechanisms are currently used in your Datadog environment, navigate to the [Ingestion Control Page][3].
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/ingestion_control_page.png" alt="Ingestion Control Page" style="width:90%;" >}}
+
+The table gives insights on ingested volumes *by service*. The Configuration column provides a first indication of the current set up. It shows: 
+- `AUTOMATIC` if the sampling rate calculated in the Datadog Agent is applied to the traces that start from the service. Read more about the specifics of [Datadog Agent ingestion logic][5].
+- `CONFIGURED` if a custom sampling rate configured in the tracing library is applied to the traces that start from the service.
+
+Click on services to see details about what sampling decision makers are used for each service, as well as what [ingestion sampling mechanisms][1] are leveraged for ingested spans' services. 
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/service_ingestion_summary.png" alt="Service Ingestion Summary" style="width:90%;" >}}
+
+## Keeping certain types of traces
+
+### Keeping entire transaction traces
+
+Ingesting entire transaction traces ensures visibility over the **end-to-end service request flow** for specific individual requests.
+
+#### Solution: Head-based sampling
+
+Complete traces can be ingested with [head-based sampling][4] mechanisms: the decision of keeping or dropping the trace is taken at the point of trace creation (trace root span), and this decision is propagated in the request context to downstream services.
+
+{{< img src="/tracing/guide/ingestion_sampling_use_cases/head_based_sampling_keep.png" alt="Head-based Sampling" style="width:100%;" >}}
+
+To decide which traces to keep and drop, the Datadog Agent computes default sampling rates for each service to apply at trace creation, based on the application traffic:
+- For low-traffic applications, a sampling rate of 100% is applied.
+- For high-traffic applications, a lower sampling rate is applied with a target of 10 complete traces per second per Agent.
+
+You can also override the default Agent sampling rate by configuring the sampling rate by service. See how to [keep more traces for specific services](#keeping-more-traces-for-high-visibility-into-specific-services-or-resources) for more information.
+
+#### Configuring head-based sampling
+
+Default sampling rates are calculated to target 10 complete traces per second per Agent. You can increase or decrease this target by configuring the Datadog Agent parameter `max_traces_per_second` or the environment variable `DD_APM_MAX_TPS`. Read more about [head-based sampling ingestion mechanisms][5].
+
+**Note:** Changing an Agent configuration impacts the percentage sampling rates for *all services* reporting traces to this Datadog Agent.
```

Comment:
It's still per a service though right? Like `service A` send 10 traces that second, `service B` send 7, all of them are still taken right? Maybe this should be clarified here?

Reply:
so line 57 should say "10 complete traces per service per second per Agent," yeah?

Reply:
Yeah